kong/controlplane-connector/components/kong.gw.manager/src/main/java/org/wso2/kong/client/KongFederatedAPIDiscovery.java

```diff
@@ -156,6 +162,17 @@ public List<API> discoverAPI() {
             api.setEndpointConfig(KongAPIUtil.buildEndpointConfigJson(endpoint, endpoint, false));
             api.setAvailableTiers(new HashSet<>(java.util.Collections.singleton(new Tier("Unlimited"))));
 
+            for (KongPlugin plugin : plugins) {
+                String pluginType = plugin.getName();
+                if (pluginType.equals("cors")) {
```

Comment:
shall we add constants, check others as well

Reply:
added constants